packages/migrations/src/migrate.ts

```diff
@@ -1,30 +1,51 @@
 #!/usr/bin/env node
 import { devConstants, web3Factory } from '@0xproject/dev-utils';
 import { logUtils } from '@0xproject/utils';
+import { Web3Wrapper } from '@0xproject/web3-wrapper';
 import { Provider } from 'ethereum-types';
 import * as yargs from 'yargs';
 
-import { runV1MigrationsAsync } from './v1/migration';
-import { runV2MigrationsAsync } from './v2/migration';
+import { runV1MigrationsAsync } from './1.0.0/migration';
+import { runV2TestnetMigrationsAsync } from './2.0.0-beta-testnet/migration';
+import { runV2MigrationsAsync } from './2.0.0/migration';
+
+import { constants } from './utils/constants';
+import { providerFactory } from './utils/provider_factory';
 
 enum ContractVersions {
     V1 = '1.0.0',
     V2 = '2.0.0',
+    V2Testnet = '2.0.0-beta-testnet',
 }
 const args = yargs.argv;
 
 (async () => {
+    const providerConfigs = { shouldUseInProcessGanache: false };
+    const provider: Provider = web3Factory.getRpcProvider(providerConfigs);
```

Comment:
Move these two lines under both the `v1` and `v2` cases. It seems duplicitous, but having it up here is miss-leading since not all cases use it.